src/Compilers/Core/Portable/CodeAnalysis.csproj

```diff
@@ -45,6 +45,7 @@
     <PackageReference Include="System.Collections.Immutable" Version="$(SystemCollectionsImmutableVersion)" />
     <PackageReference Include="System.Reflection.Metadata" Version="$(SystemReflectionMetadataVersion)" />
     <PackageReference Include="System.Text.Encoding.CodePages" Version="$(SystemTextEncodingCodePagesVersion)" />
+    <PackageReference Include="System.Threading.Tasks.Extensions" Version="$(SystemThreadingTasksExtensionsVersion)" />
```

Comment:
You could update the `PerformanceSensitiveAttribute`'s docs to include a proper `<see cref="ValueTask" />` tag now 😉 

https://github.com/dotnet/roslyn/blob/682208901aee33a1d322bc5fdf488106e3a3cfb9/src/Compilers/Core/Portable/InternalUtilities/PerformanceSensitiveAttribute.cs#L82